FastSimulation/Muons/plugins/FastTSGFromL2Muon.cc

```diff
@@ -49,8 +49,9 @@ FastTSGFromL2Muon::beginRun(edm::Run const& run, edm::EventSetup const& es)
   edm::ParameterSet regionBuilderPSet = 
     theConfig.getParameter<edm::ParameterSet>("MuonTrackingRegionBuilder");
   edm::ConsumesCollector iC  = consumesCollector();
-  theRegionBuilder = new MuonTrackingRegionBuilder(regionBuilderPSet,theService,iC);
```

Comment:
@BenjaminRS 

Hi Benjamin

I assume you know what you're doing, cause I'm not familiar with this code. :-)

Would you mind removing theService from this class.
It's not used anymore after your change.

Thanks

Lukas